ssl/ssl_conf.c

```diff
@@ -107,6 +107,7 @@ static void ssl_set_option(SSL_CONF_CTX *cctx, unsigned int name_flags,
                            unsigned long option_value, int onoff)
```

Comment:
Isn't the `unsigned long` here for `option_value` problematic? At least it will be as soon as we have a config value using `SSL_OP_BIT(32)` or above

Reply:
excellent catch!  changed to uint64_t